{{ cookiecutter.project_slug }}/Makefile

```diff
@@ -56,13 +56,16 @@ export PRINT_HELP_PYSCRIPT
 export START_DOCKER_CONTAINER
 export PYTHONPATH=$PYTHONPATH:$(PWD)
 export PROJECT_NAME={{ cookiecutter.project_slug }}
-export IMAGE_NAME=$(PROJECT_NAME)-image
-export CONTAINER_NAME=$(PROJECT_NAME)-container
+export MODE=dev ## dev or release
```

Comment:
Comments with `#` will need to be one line above that line
When I ran it the `export MODE=dev ## comment` was interpreted as `MODE= dev `
I mean, there was a blank in the back.

Reply:
👍